install-instructions.php

```diff
@@ -87,37 +87,39 @@
 			      </div>
-			      <div class="col-md-9">
-			        Offers live USB stick, liveCD and preload ISO images as well as images for VMware (ESX<em>i</em>), VirtualBox, IBM SmartCloud, XEN, Oracle VM and KVM-based hypervisors, Microsoft Hyper-V servers and OpenStack clouds.
+			      <div class="col-md-9 vmlist">
+			        ownCloud-in-a-box offers live USB stick, liveCD and preload ISO images as well a wide variety of VM images.
 			      </div>
 			    </div>
 			    <div class="row">
-                              <div class="col-md-3">
+                              <div class="col-md-3 vmlist">
                                 <a target="_blank" style="margin: 0 14px 0 7px;" href="https://bitnami.com/stack/owncloud"><img width=113 src="<?php echo get_template_directory_uri(); ?>/assets/img/clouds/bitnami.png"></img></a>
                               </div>
-			      <div class="col-md-9">
-			    Offers direct deployment to Google Cloud, Amazon AWS and Microsoft Azure, local install, Docker images, VMware images and installers for Linux and Mac.
+			      <div class="col-md-9 vmlist">
+				Offers direct deployment to Google/Amazon/Microsoft clouds, Docker and VM images and installers.
 			      </div>
 			    </div>
 			    <div class="row">
-                              <div class="col-md-3">
+                              <div class="col-md-3 vmlist">
 			        <a target="_blank" style="margin: 0 14px 0 7px;" href="https://jujucharms.com/owncloud/"><img width=113 src="<?php echo get_template_directory_uri(); ?>/assets/img/clouds/juju.png"></img></a>
 			      </div>
-			      <div class="col-md-9">
+			      <div class="col-md-9 vmlist">
 			        This charm installs Apache2, Apache-Mod-PHP5, and configures ownCloud with SQLite as a standalone server on an Ubuntu system.
 			      </div>
 			    </div>
 			    <div class="row">
-                              <div class="col-md-3">
+                              <div class="col-md-3 vmlist">
 			        <a target="_blank" style="margin: 0 14px 0 7px;" href="https://www.en0ch.se/owncloud_/pre-configured-owncloud-installaton/"><img width=113 src="<?php echo get_template_directory_uri(); ?>/assets/img/clouds/en0ch.png"></img></a>
 			      </div>
-			      <div class="col-md-9">
-			        This pre-configured VM runs on Ubuntu 14.04 with Apache and MySQL, and is <a href="https://www.en0ch.se/machine-setup/" target="_blank">fully configured from start</a>. Updates are managed through a built in script, and could be setup as a cron job for automatic updates. 
+			      <div class="col-md-9 vmlist">
+			        <a href="https://www.en0ch.se/machine-setup/" target="_blank">Pre-configured Ubuntu 14.04 Vmwith Apache and MySQL</a>. Managers updates with script, can be automated.
 			      </div>
```

Comment:
rewrote some stuff and I guess the diff tool just puked all over it ;-)

But it looks fine, so... ship it :package: